lunavl/sdk/faceengine/facedetector.py

```diff
@@ -313,21 +339,30 @@ def detect(
 
         return res
 
-    def redetectOne(self):
+    def redetectOne(self, image: ImageForRedetection):
```

Comment:
I thin, that  we should support  following interface for usability:
 `ImageForRedetection`
 `VLImage` + `Rect`
 `VLImage`  + detection

Reply:
Also need @CoreExceptionWarp

Reply:
done